v2/devices/Spacewar.yml

```diff
@@ -0,0 +1,193 @@
+name: "Nothing Phone 1"
+codename: "Spacewar"
+formfactor: "phone"
+aliases: []
+doppelgangers: []
+user_actions:
+  confirm_model:
+    title: "Confirm your model"
+    description: "Please check that your device is a Nothing Phone 1 (Spacewar)."
+  downgrade_android:
+    title: "Downgrade Android"
+    description: "Stock Android 13 is required to install Ubuntu Touch. This device has a rollback protection that needs to be disabled first in order to downgrade. Click on More... for the downgrade process."
+    link: "https://xdaforums.com/t/advanced-bypassing-rollback-protection-to-downgrade-the-os.4511501"
+  confirm_os:
+    title: "Confirm OS version"
+    description: "Your device must be running NothingOS version of Android 13 (NothingOS 2.0.5) before installing Ubuntu Touch. Any other type of ROM (stock or custom) won't work! Click below on More... to download NOS 2.0.5 and follow the steps on the downgrade section."
+    link: "https://sourceforge.net/projects/nothingphone-1-rom/files/FastbootROM"
```

Comment:
`https://sourceforge.net/projects/nothingphone-1-rom/files/FastbootROM/Spacewar_FastbootROM_v2.0.5.7z/download` would be a better link but it's not a big deal, we should probably change how the links look

Reply:
I should be able to test the installation sometime next week if you don't find anyone by then, found a cheap used one. I'll keep you in mind for UDFPS testing.

LEDs would be a nice addition to the port, yeah :D